### PR TITLE
Only include exported members in docs

### PR DESCRIPTION
**Pull request by KnorpelSenf**
# Description

Updates `typedoc` to a beta version in order to remove clutter from the generated docs by reducing the displayed entities to those that `telegraf` actually `export`s.

Also, switching to the `minimal` theme gives the docs a much better overview.

## Type of change

Config update

# How Has This Been Tested?

`npm run build:docs`

# Checklist:

- [X] I have performed a self-review of my own code
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings

### CODE REVIEW

File: typedoc.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "name": "telegraf.js"
```
Comment (wojpawlik) [inline typedoc.json:2]: Why is this needed? Typedoc seems to infer `"telegraf"`

File: package.json [hunk 2]
```diff
@@ -31,7 +31,7 @@
   "scripts": {
     "prepare": "npm run build && mjs-entry src/index.ts > lib/index.mjs",
     "build": "tsc --build docs/examples",
-    "build:docs": "typedoc",
+    "build:docs": "typedoc src/index.ts --theme minimal",
```
Comment (wojpawlik) [inline package.json:34]: Let's discuss the theme later, once we can actually use the docs in prod.
```suggestion
    "build:docs": "typedoc src/index.ts",
```

**Review by wojpawlik — CHANGES_REQUESTED** [review]

File: typedoc.json (hunk 1 above)
Comment (KnorpelSenf) [reply typedoc.json:2]: It's already pretty inconsistent with Telegraf vs. telegraf.js. Let's not introduce a third name

Comment (wojpawlik) [reply typedoc.json:2]: Fair enough, just move this to `tsconfig`

Comment (wojpawlik) [reply typedoc.json]: `tsconfig.json` not `package.json`

Comment (KnorpelSenf) [reply typedoc.json]: Oops

**Review by wojpawlik — APPROVED** [review]
Why did you change `bash` to `shellscript` and `plaintext` to `text`?

**Comment by KnorpelSenf** [comment]
> Why did you change `bash` to `shellscript` and `plaintext` to `text`?

`typedoc` does not support `bash` or `plaintext`. Run `typedoc --help` to find out what kind of markdown syntax highlighting is supported.

The change removes five warnings.
